verification/config/base.env

```diff
@@ -0,0 +1,25 @@
+#VERIFICATION__SERVER__ADDR=0.0.0.0:8043
```

Comment:
I think most env files taking the token without '#'
https://docs.docker.com/compose/environment-variables/#the-env-file
and `#string` is actually a comment

Reply:
You are right, but the main idea here was just to have a file with all configuration variables and their default values. So that users could copy required variables if they would like to use envvars to customize their setup. 

If we do not comment those values, however, they will overwrite values written in `basic.toml` and no changes to `basic.toml` will take a place when the default docker-compose is running. In order to avoid that I commented values in `base.env`

Reply:
I've added some comments describing that into `base.env`